scripts/download_munge_wqp.R

```diff
@@ -24,7 +25,15 @@ id_from_status <- function(status.file){
   readLines(status.file, n=1L)
 }
 
-munge_wqp <- function(wqp.data){
-  # dplyr
-  
+munge_secchi <- function(data.in){
+  rename(data.in, Date=ActivityStartDate, value=ResultMeasureValue, units=ResultMeasure.MeasureUnitCode, wqx.id=MonitoringLocationIdentifier) %>% 
+    select(Date, value, units, wqx.id) %>% 
+    mutate(secchi = ifelse(units=='in', value*0.0254, ifelse(units=='ft', value*0.3048, ifelse(units=='cm', value*0.01, value)))) %>% 
+    filter(!is.na(secchi)) %>% 
```

Comment:
Couldn't get it to work w/ a bunch of different attempts. Giving up and using that nasty syntax I have above.